spec/models/interactions_spec.rb

```diff
@@ -7,4 +7,59 @@
   it { should validate_presence_of :category}
   it { should validate_presence_of :response}
 
+
+  scenario "one_of_three_interactions returns error if sentence is blank" do
+    interaction = Interaction.create(
+      category: "Sentence",
+      sentence: '',
+      response: Faker::Lorem.sentence
+    )
+    expect(interaction.one_of_three_interactions).to eq(["You need to fill the sentence field", "You need to fill the sentence field"])
+  end
+
+  scenario "one_of_three_interactions returns error if keyword1 is blank choosing category keyword" do
+    interaction = Interaction.create(
+      category: "Keyword",
+      sentiment: "Positive",
+      keyword1: '',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.